web/src/shared/utilities/sanitiseValue.tsx

```diff
@@ -0,0 +1,10 @@
+const sanitiseValue = (value: unknown): string => {
```

Comment:
This is a plain typescript (`ts`) file. The `tsx` file extension should be reserved for files that render JSX.